### PR TITLE
Fixes Issue #34 

### PR DESCRIPTION
**Pull request by tmadruga**
- Changed the location of filtering to be in scatterplots because material felt redundant from Module A
- Changes sentence structure for clarity

### CODE REVIEW

**Comment by raskutti** [comment]
@tmadruga please fix CLA

**Comment by lindseyc** [comment]
@googlebot I consent.

File: ac1/_sources/scatter_plots_and_correlation/correlation_and_filtering.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Correlation and Filtering
+=========================
+
+As you learned in Module A, filtering is a useful technique to help make sense
+of a large dataset. Filtering data helps identify the similarities and
+differences between groups and describe the relationships between variables.
+Recall that a filter is a way of selecting a subset of rows based on a set of
+column conditions. For example, if you have population data over the past ten
+years for each state, you could use a filter to analyze data only for Florida.
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/correlation_and_filtering.rst]: I would include back "or only for the year 2017".

File: ac1/_sources/scatter_plots_and_correlation/correlation_and_filtering.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Often, you can only see important differences or trends by filtering. This is
+even the case when finding the correlation coefficient that best reflects the
+data. This is best explained through an example.
+
+Imagine that your friend is a `personal trainer who has implemented a new
+strength training regime <https://drive.google.com/open?id=1kzEQ-JCgr0RLCb0ojl1HldF0irO_UzQO0PtxsjBRFtU>`_
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/correlation_and_filtering.rst:22]: This file is private.

**Review by raskutti — APPROVED** [review]
Looks great! Please address the minor comments @tmadruga

**Comment by raskutti** [comment]
Looks good. Sorry for the delay, but there are now merge conflicts. Once these are resolved, I will merge the change.

**Comment by raskutti** [comment]
@tmadruga

**Comment by tmadruga** [comment]
@raskutti fixed merge conflict.

**Pull request merged**